protocol/x/subaccounts/keeper/negative_tnc_subaccount.go

```diff
@@ -9,28 +9,30 @@ import (
 )
 
 // GetNegativeTncSubaccountSeenAtBlock gets the last block height a negative TNC subaccount was
-// seen in state. Note this defaults to 0 if it has not been set.
+// seen in state and a boolean for whether it exists in state.
 func (k Keeper) GetNegativeTncSubaccountSeenAtBlock(
 	ctx sdk.Context,
-) uint32 {
+) (uint32, bool) {
 	store := ctx.KVStore(k.storeKey)
 	return k.getNegativeTncSubaccountSeenAtBlock(store)
 }
 
 // getNegativeTncSubaccountSeenAtBlock is a helper function that takes a store and returns the last
-// block height a negative TNC subaccount was seen in state.
+// block height a negative TNC subaccount was seen in state and a boolean for whether it exists in state.
```

Comment:
As an FYI, updated this function to avoid defaulting to 0 so that withdrawals aren't blocked when block height is less than `WITHDRAWAL_AND_TRANSFERS_BLOCKED_AFTER_NEGATIVE_TNC_SUBACCOUNT_SEEN_BLOCKS` (happens after chain restarts from genesis, unit tests, etc).